docs/csharp/fundamentals/coding-style/coding-conventions.md

```diff
@@ -153,6 +153,89 @@ Good layout uses formatting to emphasize the structure of your code and to make
 - Use parentheses to make clauses in an expression apparent, as shown in the following code.
 
   :::code language="csharp" source="./snippets/coding-conventions/program.cs" id="Snippet2":::
+  
+## Place the using directives outside the namespace declaration.
+
+When the using-import directive is outside the namespace everyone understands what it’s doing.  When it’s inside it becomes context-sensitive, and therefore, ambiguous.
```

Comment:
Let's word this differently:

```suggestion
When a `using` directive is outside a namespace declaration, that imported namespace is its fully qualified name. That's more clear. When the `using` directive is inside the namespace, it could be either relative to that namespace or it's fully qualified name. That's ambiguous.
```